components/collaboration/UserResponse.tsx

```
@@ -153,34 +153,35 @@ const UserResponse: FC<UserResponseProps> = ({
-                  <Text
+          {application?.answersFromInfluencer
+            ? influencerQuestions &&
+              application?.answersFromInfluencer.map((answer, index) => {
+                return (
+                  <View
                     style={{
-                      fontSize: 16,
+                      flexDirection: "column",
+                      gap: 10,
                     }}
                   >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)